bin/repl.js

```diff
@@ -36,7 +36,7 @@ const proc = spawn('ssc', args, {
 })
 
 let nextId = 0
-let socket = null
+let sock = null
```

Comment:
I think `sock` and `socket` being different things, in the same code base, may turn out to be a bit confusing for future maintenance/review purposes.

Might we consider `conn` as the variable name for socket connections?